### PR TITLE
FCBH-2435 Plan translation (ONE>NIV, maybe others)

### PR DESCRIPTION
**Pull request by mgonzalezs**
# Description
- Fix order by adding the 'order_column' while the pl/an is being translated, without this the order of the items is not being inserted correctly causing conflicts on the translated view.

## Issue Link
[FCBH-2435](https://fullstacklabs.atlassian.net/browse/FCBH-2435) 

## How Do I QA This
- Open the app with the FCBH-2435 backend branch seletec.
- Navigate to the ONE 2021 plan
image
- Click on translate -> English -> NIV
- Verify you get the correct conent

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed locally.

### CODE REVIEW

**Review by alburdette619 — APPROVED** [review]

**Review by bradflood — CHANGES_REQUESTED** [review]
@mgonzalezs UsersController doesn't seem to be related to this PR.  I suspect there were just some formatting changes you wanted to add. If so, could you please remove from this PR and we can address UsersController separately

**Comment by mgonzalezs** [comment]
@bradflood I can't commit with lint errors. Not sure how Casey committed and merged the previous PR, was merged before the approval. 
image. 

These are the rules 
image

These are the commands

image

If we have lint errors we should run `composer cs-fix` or fix them manually. 

PD: I know we don't have CI yet implemented to prevent this.

**Comment by bradflood** [comment]
@mgonzalezs  I will follow-up with Casey on the linting process, but won't hold up this PR.

**Pull request merged**
